XamarinCommunityToolkit/Views/AvatarView.shared.cs

```diff
@@ -132,7 +132,7 @@ public FontAttributes FontAttributes
 		protected override void OnControlInitialized(Frame control)
 		{
 			IsClippedToBounds = true;
-			MainLayout.Children.Add(Label, new Rectangle(0, 0, 1, 1), AbsoluteLayoutFlags.All);
+			MainLayout.Children.Add(Label, new Rectangle(0.5, 0.5, -1, -1), AbsoluteLayoutFlags.PositionProportional);
```

Comment:
#442 Centralize label

Reply:
Weird

```
HorizontalTextAlignment = TextAlignment.Center,
VerticalTextAlignment = TextAlignment.Center,
```

Seems these guys don't work as expected

Reply:
Why did you choose AbsoluteLayout as the main layout? are there any performance preferences over the grid?
```xml
<Grid>
        <Label HorizontalOptions="CenterAndExpand" VerticalOptions="CenterAndExpand" />
        <Image
            HorizontalOptions="CenterAndExpand"
            VerticalOptions="CenterAndExpand" />
    </Grid>
```

Reply:
Yes, as I know AbsoluteLayot has the best performance in comparison with all other layouts in XF